tests/release/pipelines/e2e-test-default-with-deployment.go

```diff
@@ -13,10 +15,9 @@ import (
 	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
 	tektonutils "github.com/redhat-appstudio/release-service/tekton/utils"
 	corev1 "k8s.io/api/core/v1"
-	"os"
 )
 
-var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release-pipelines", "withDeployment", "HACBS"), func() {
+var _ = framework.ReleaseServicesSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release-services", "withDeployment", "HACBS"), func() {
```

Comment:
Pls. use "release-service" instead of "release-services" and "ReleaseServiceSuiteDescribe" instead of "ReleaseServicesSuiteDescribe"